src/main/java/org/frc/team4048/rosnetworktables/NtToRTopic.java

```diff
@@ -26,27 +31,34 @@ protected NtToRTopic(NetworkTableInstance ntInst, Subscriber ntSubscriber,Publis
     public void start() {
         // Register the NT subscriber to messages
         ntInst.addListener(ntSubscriber, EnumSet.of(NetworkTableEvent.Kind.kValueAll),
-// TODO this is bad practice needs to be look at
+                // TODO this is bad practice needs to be look at
```

Comment:
What is "bad practice" here?

Reply:
The cast to (T). It's because the way the NT API is built.